roles/ngi-rnaseq/tasks/main.yml

```diff
@@ -6,11 +6,21 @@
        force=yes
 
 - name: Copy NGI-RNAseq config
-  template: src="ngi-rnaseq.config.j2" dest="{{ ngi_pipeline_conf }}/ngi-rnaseq_{{ item.site }}.config"
+  template: src="{{ ngi_rnaseq_dest }}/conf/uppmax.config" dest="{{ ngi_pipeline_conf }}/ngi-rnaseq_{{ item.site }}.config"
   with_items:
   - { site: "sthlm", proj: "{{ ngi_pipeline_sthlm_delivery }}" }
   - { site: "upps", proj: "{{ ngi_pipeline_upps_delivery }}" }
 
+- name: Apply NGI specific config changes
+  lineinfile: dest="{{ ngi_pipeline_conf }}/ngi-rnaseq_{{ item.site }}.config"
```

Comment:
This should work. Even had a discussion if it is ok to place them at the end of the file.